cpp/log/cert.cc

```diff
@@ -887,10 +963,23 @@ bool ValidateRedactionSubjectAltNames(STACK_OF(GENERAL_NAME) *
 }  // namespace
 
 
+util::Status Cert::SubjectAltNames(vector<string>* dns_alt_names) const {
+  ScopedGENERAL_NAMEStack subject_alt_names(
+      static_cast<STACK_OF(GENERAL_NAME)*>(X509_get_ext_d2i(
+          x509_.get(), NID_subject_alt_name, nullptr, nullptr)));
+
+  if (subject_alt_names) {
+    return ExtractSubjectAltNames(subject_alt_names.get(), dns_alt_names);
+  }
+
+  return util::Status::OK;
```

Comment:
I think it's quite valid for a cert to have no SANs so returning OK with the vector set empty should be good, no?